### DISCUSSION TITLE
Could not parse the incoming JSON

### DISCUSSION THREAD

**Discussion by VelSankar-DT** [apache/camel-karavan#1166] [Q&A]
**Body:**
{
"name": "Curaçao"
}

**Neof4j API:**
http://{{neo4jHost}}:{{neo4jPort}}/db/{{neo4jDatabaseName}}/tx/commit

**Request body for API:**
{
"statements": [
{
 "statement": "MERGE (node:Country {name: '${jq(.name)}'})"
}]
}

**Error:**
{"results":[],"errors":[{"code":"Neo.ClientError.Request.InvalidFormat","message":"Could not parse the incoming JSON"}]}

**Notes:**
Other country names are working properly and its working fine in postman

**Comment by davsclaus** [comment]
Its hard to help you as your description above is not informative. Tell us which Camel version you use, and if possible show the camel route (yaml) that you have build with karavan

**Comment by davsclaus** [comment]
closing as user did not respond
